src/lib/storage/table.hpp

```diff
@@ -72,6 +72,9 @@ class Table : private Noncopyable {
 
  protected:
   // Implementation goes here
+  ChunkOffset _target_chunk_size;
```

Comment:
Not initialized. As you don't delete the default constructor of table, one might create a table with an arbitrary chunk size (see lecture).